Mathlib/RingTheory/Subsemiring/Basic.lean

```diff
@@ -1074,7 +1070,9 @@ end Subsemiring
 namespace RingHom
 
 variable [NonAssocSemiring T] {s : Subsemiring R}
+
```

Comment:
I thought we just removed these?

Reply:
Sorry that's a probably a bad merge on my part